src/app/public/modules/needs-attention/needs-attention.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="sky-elevation-1-bordered sky-padding-even-xl">
+  <h2 class="sky-font-heading-4">{{ 'Needs Attention' | skyLibResources }}</h2>
+  <sky-fluid-grid [gutterSize]="0">
+    <sky-row>
+      <sky-column
+        *ngFor="let item of items; let isLast = last"
+        screenXSmall="12"
+        [screenSmall]="items.length > 6 ? 6 : 12"
+      >
+        <ng-template #link>
+          <sky-icon
+            class="needs-attention-item-icon"
+            icon="chevron-right"
+          ></sky-icon>
+          <span class="needs-attention-item-label sky-padding-horizontal-sm">{{
```

Comment:
Can we use the Key Info component (with sky-font-display-4 on the value) here to get the correct styling?